toonz/sources/tnztools/geometrictool.cpp

```diff
@@ -380,6 +402,8 @@ class PrimitiveParam {
     m_capStyle.setQStringName(tr("Cap"));
     m_joinStyle.setQStringName(tr("Join"));
     m_miterJoinLimit.setQStringName(tr("Miter:"));
+    m_snap.setQStringName("Snap");
```

Comment:
Not sure if this is intentional, but you might be missing the `tr()` here for localization.